### PR TITLE
Modernize (Convert to roxygen2)

### LINKED ISSUE #1: Rework Package to use roxygen
Just putting this out there.  It looks like this packages was put together the "old fashioned" way. If there isn't any resistance I will rework it so it uses roxygen2

### PR DESCRIPTION
**Pull request by JackStat**
Take a look at this at lets see if this resolves #1.

### CODE REVIEW

**Comment by farmdawgnation** [comment]
Hey Tyler, sorry this has been sitting for as long as it has. For some reason I thought @nelprin was going to scan it over. Since it appears to be unclaimed I'll add it to my list of reminders and try to get to it this week.

Thanks for the contribution. :)

**Comment by JackStat** [comment]
Thanks Matt.  I would like to get and do some more but I want to see if these changes stick first

File: domino/R/domino.upload.R [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+#' did to the project code in recent update. Ex. \code{"updated project files"}
+#' 
+#' @keywords upload
+#' @export
+
+domino.upload <- function(commitMessage) {
+  if(missing(commitMessage)) {
+    domino.runCommand("upload", domino.OK, "Uploading project data failed.")
+  } else {
+    cmd = paste("upload -m \"", commitMessage, "\"", sep="");
```
Comment (farmdawgnation) [inline domino/R/domino.upload.R:18]: This isn't escaping quotes in the commit message. That's probably going to bite someone.

**Comment by farmdawgnation** [comment]
Hammered on it a bit and was able to find a bit of brokenness with login - but nothing related to these changes. Let's push them in and revisit.

**Pull request merged**